nixos/modules/services/networking/sslh.nix

```diff
@@ -56,9 +60,9 @@ in
       };
 
       listenAddress = mkOption {
```

Comment:
```suggestion
      listenAddresses = mkOption {
```

Reply:
this breaks backward compatability, so I'd like to avoid this.

Reply:
Yes, but it is not correct to call it listenAddress and have it be a list of multiple addresses. Therefore, the rename is inevitable unless there is a way to listen on both with a single listenAddress specification.

Reply:
I agree with Graham, backwards compatibility is somewhat broken anyways by the type change and its best to be explicit about it. Failing at evaluation time with an obvious error and an obvious fix is a relatively cheap backwards-incompatible change from a user perspective.

Reply:
Hmm I forgot to mark this conversation as solved. I had already renamed the option with mkRenamedOptionModule so it's user friendly.